tree/dataframe/src/RDataFrame.cxx

```diff
@@ -455,8 +479,175 @@ cols = df.Filter("x > 10").AsNumpy(["x", "y"])
 print(cols["x"], cols["y"])
 ~~~
 
+##  <a name="transformations"></a>Transformations
+### <a name="Filters"></a> Filters
+A filter is defined through a call to `Filter(f, columnList)`. `f` can be a function, a lambda expression, a functor
```

Comment:
Most people will use the jitted version, without the explicit column list. Isn't this a little confusing if you've seen RDF before, but you remember the `df.Filter("x > 0")` call?

Reply:
I agree, I think this part of the docs was written before we had jitting. Will update to mention both overloads

Reply:
done